packages/pasteup/README.md

```diff
@@ -1,79 +1,230 @@
 # Pasteup
 
-Guardian design tokens. These tokens are intended to be used in conjunction with a CSS-in-JS library
-such as [Emotion](https://emotion.sh).
+The Guardian Design Tokens are intended to be used in conjunction with a CSS-in-JS library such as [Emotion](https://emotion.sh). 
 
-**Alpha package - under construction 🚧**
+**Under construction 🚧**
```

Comment:
Can we make it explicit that the API is still likely to change? Really we don't want anyone trying to use this package (except us) until we get to version 1.0.0 (we're currently at `1.0.0-alpha.1`)

Reply:
Done